sechub-cli/src/daimler.com/sechub/testutil/fileTestHelpers.go

```diff
@@ -32,7 +32,7 @@ func CreateTestDirectory(dir string, mode os.FileMode, t *testing.T) {
 	if err != nil {
 		t.Fatalf("Cannot create folder. Error: %q", err)
 	}
-	log.Printf("Folder created: %q", dir)
+	fmt.Printf("Folder created: %q", dir)
```

Comment:
I would leave it for now, because on the the main goals was to avoid dependencies to external libraries.

Reply:
In case you do not want to use an external library, you could create a small helper function which decides what output should go where. The benefit would be that you have one function/method which is the same across the entire code base. In addition, you can change the logging implementation in the future, without having to touch every piece of code you use it in.

A pseudo-go-code example:

~~~
const (
 TRACE
 DEBUG
 INFO
 WARNING
 ERROR
)

type Level int

func myLog(severity Level, message string) {
 switch level {
   case ERROR:
     log.Println(message)
     return 
   ...
   case INFO:
     fmt.Println(message)
     return
   ...
 }
}
~~~
See: https://stackoverflow.com/a/52089219

In your code you can use it like this:

~~~
myLog(INFO, "something happened")
myLog(ERROR, "something went wrong")
~~~